mediaire_toolbox/queue/daemon.py

```diff
@@ -54,8 +54,15 @@ def __init__(self,
         self.stopped = False
         self.processing_t_id = None
 
-        signal.signal(signal.SIGINT, self.exit_gracefully)
-        signal.signal(signal.SIGTERM, self.exit_gracefully)
+        try:
+            signal.signal(signal.SIGINT, self.exit_gracefully)
```

Comment:
These lines can only be executed if you instantiate the daemons in the main execution thread, but that is not the case in md_cloud_compute